README.md

```diff
@@ -65,10 +65,10 @@ module "cloudwatch_logs" {
   source = "coralogix/aws/coralogix//modules/cloudwatch-logs"
 
   coralogix_region   = "Europe"
-  CustomDomain       = "https://<your custom doamin>/api/v1/logs"
+  custom_domain       = "https://<your custom doamin>/api/v1/logs"
```

Comment:
If you have coralogix_region configured no need for custom_domain and vise-versa